packages/migrate/src/__tests__/__snapshots__/DbPull.test.ts.snap

```diff
@@ -66,6 +66,121 @@ model jobs {
 }
 
 
+// introspectionSchemaVersion: NonPrisma,
+`;
+
+exports[`cockroachdb basic introspection --url 2`] = `
+datasource db {
+  provider = "postgresql"
+  url      = "postgresql://prisma@localhost:26257/tests"
```

Comment:
The value here will be `cockroachdb` instead of `localhost` for Buildkite so we can normalize this here 
https://github.com/prisma/prisma/blob/main/packages/sdk/src/utils/jestSnapshotSerializer.js#L73:L75

Reply:
Thanks, this was the sanitization part I was missing yesterday!

Reply:
@Jolg42 still, we need a `.github/setup-cockroachdb.sh` file. I'll try to write one, but I'm not that familiar with cockroachdb